aws-sdk-resources/lib/aws-sdk-resources/services/s3/file_downloader.rb

```diff
@@ -135,31 +133,43 @@ def map_files(slice, dir, mode)
         end
       end
 
-      def get_range(chunks)
+      def clean_up_parts(parts)
```

Comment:
Make sure this can handle a `nil` value for `parts`, since an ensure block is going to call this method.